project_variables.go

```diff
@@ -132,6 +133,7 @@ type CreateProjectVariableOptions struct {
 	Description      *string            `url:"description,omitempty" json:"description,omitempty"`
 	EnvironmentScope *string            `url:"environment_scope,omitempty" json:"environment_scope,omitempty"`
 	Masked           *bool              `url:"masked,omitempty" json:"masked,omitempty"`
+	Hidden           *bool              `url:"hidden,omitempty" json:"hidden,omitempty"`
```

Comment:
From the [docs](https://docs.gitlab.com/ee/api/project_level_variables.html#create-a-variable:~:text=masked.%20Default%3A%20false-,masked_and_hidden,-boolean) it looks like it's called `masked_and_hidden`. Did `hidden` work for you?

Reply:
API in docs perspective `masked_and_hidden` is consistent name. I have not tested this against the api.

Reply:
There also might be validation error as masked and masked_and_hidden should be mutually exclusive as per UI

Reply:
Well, we need to support in go-gitlab exactly with what the upstream API supports. 

> I have not tested this against the api.

I highly recommend doing this :)

Reply:
The name is changed as per api documentation.

>  > I have not tested this against the api.
>
> I highly recommend doing this :)

Still not tested as its not easy to setup GitLab by any means